packages/admin/cms-admin/src/userPermissions/user/UserPageToolbar.tsx

```diff
@@ -38,7 +53,25 @@ export const UserPermissionsUserPageToolbar = ({ userId }: { userId: string }) =
             </ToolbarTitleItem>
             <ToolbarFillSpace />
             <ToolbarActions>
-                <StartImpersonationButton userId={userId} />
+                <CrudMoreActionsMenu
```

Comment:
This could more actions menu could be empty if a user isn't allowed to impersonate other users. Is this handled correctly?

Reply:
118c22839806f14a35d3e60bf069bd649b7db1f4